test/cypress/e2e/search.spec.js

```diff
@@ -228,7 +215,8 @@ describe('Search', () => {
    */
   it('Search for nonexistent item - shows no results plug', () => {
     cy.visit(urls.dashboard);
-    cy.get(ss.searchInput).click().type('321321{enter}');
-    cy.get(ss.emptyResultsMessage).should('have.text', 'No results');
+    cy.get(ss.searchIcon).click();
+    cy.get(ss.searchInput).type('321321');
+    cy.get(ss.searchMessage).eq(0).should('have.text', 'No results found.');
```

Comment:
This test was made for No Results Found screen. There is no way to get there now. It now duplicates preceding one. Test should be removed.

Reply:
fixed